ChangeLog.md

```diff
@@ -1,3 +1,39 @@
+# ooniprobe-android 1.2.0 [2017-09-xx]
```

Comment:
@lorenzoPrimi remember to change the release date

Reply:
Since 1.2.0 was already cut from what I can see. The real release is actually going to be 1.2.1 and this should be listed in the changelog as a YANKED version.